google_speech/__init__.py

```diff
@@ -140,6 +148,18 @@ def play(self, sox_effects=()):
       for preloader_thread in preloader_threads:
         preloader_thread.join()
 
+  def save(self,path):
+    """ Saves to an MP3 file """
+
+    segments,preloader_threads = self.__buildSegments__()
```

Comment:
I don't think you need to use the preloader threads to save to a file, they are only useful to eliminate latency when playing in real time.